oauth-rest/oauth-resource-server/src/test/java/com/baeldung/resource/ImplicitFlowLiveTest.java

```diff
@@ -42,7 +42,7 @@ private String obtainAccessToken(String clientId, String username, String passwo
 		loginParams.put("client_id", clientId);
 		loginParams.put("response_type", "token");
 		loginParams.put("redirect_uri", REDIRECT_URL);
-		loginParams.put("scope", "oidc read write");
```

Comment:
should this be replaced with openid like in the .ts file?

Reply:
@lor6 , we added openid to the ts so as to obtain id_token along with access token. This was done for GET on LogoutEndpoint to work.

The tests here were failing with this error:
`error=invalid_request&error_description=Invalid+scopes%3A+oidc+read+write`

So i removed the oidc scope; we still need read and write scopes for our Foo API. openid is not required here since there is no need of id_token in these tests.